tests/func/test_data_cloud.py

```diff
@@ -101,11 +101,11 @@ def _should_test_ssh():
     if os.getenv("DVC_TEST_SSH") == "true":
         return True
 
-    # FIXME: enable on windows
-    if os.name == "nt":
-        return False
-
     try:
+        if os.name == "nt":
+            check_output(
```

Comment:
Why are we running it twice? It is still raising the error, so should_test returns False. This is actually one of very early tests, that has a drawback of not raising `SkipTest` exception and instead relying on `if should_test`. You could see that ssh tests are still not run by looking at coverage reports on appveyor and travis. On travis:
```
dvc/remote/ssh/__init__.py        81      7    91%
dvc/remote/ssh/connection.py     175     21    88%
```
on appveyor
```
dvc\remote\ssh\__init__.py        81     44    46%
dvc\remote\ssh\connection.py     175     99    43%
```

Reply:
Btw, if your patch is not yet ready for review, please feel free to use `[WIP]` prefix. :slightly_smiling_face:

Reply:
@efiop  Yes. I know about that. Still in progress of investigating